ipaserver/install/adtrust.py

```diff
@@ -358,10 +362,11 @@ def install_check(standalone, options, api):
     realm_not_matching_domain = (api.env.domain.upper() != api.env.realm)
 
     if realm_not_matching_domain:
-        print("WARNING: Realm name does not match the domain name.\n"
-              "You will not be able to establish trusts with Active "
-              "Directory unless\nthe realm name of the IPA server matches its "
-              "domain name.\n\n")
+        logger.info_cmd(
```

Comment:
For this kind of stuff it should be possible to increase the log level to WARNING.

Reply:
I think this should be an extra commit.